content/docs/reference/microcontrollers/swan.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Blues Wireless Swan"
+weight: 3
+---
+
+The [Swan](https://blues.io/products/swan/) is an ARM development board based on the ST Micro [stm32l4r5zi](https://www.st.com/resource/en/datasheet/stm32l4r5zi.pdf) SoC.
```

Comment:
I think the following links are good to go along with the other pages.

https://www.st.com/en/microcontrollers-microprocessors/stm32l4r5zi.html